phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -5882,6 +5912,14 @@ public void setSchemaVersion(String schemaVersion) {
         public void setStreamingTopicName(String streamingTopicName) {
             this.streamingTopicName = streamingTopicName;
         }
+
+        public Long getMaxLookbackAge() {
```

Comment:
So here max lookback age is a Long, and we have some logic dealing with it being null at, e.g line 5578...but elsewhere like on PTable we're assuming it's a long that can't be null.